src/components/enter-password/tests/enter-password-service.test.ts

```diff
@@ -0,0 +1,78 @@
+import { describe } from "mocha";
+import { enterPasswordService } from "../enter-password-service";
+import { ApiResponseResult } from "../../../types";
+import { EnterPasswordServiceInterface, UserLoginResponse } from "../types";
+import { expect } from "chai";
+import { Request } from "express";
+import { Http } from "../../../utils/http";
+import { sinon } from "../../../../test/utils/test-utils";
+import { API_ENDPOINTS } from "../../../app.constants";
+import { createMockRequest } from "../../../../test/helpers/mock-request-helper";
+import { SinonStub } from "sinon";
+
+describe("enter-password-service", () => {
+  let req: Partial<Request>;
+  const httpInstance = new Http();
+  const service: EnterPasswordServiceInterface =
+    enterPasswordService(httpInstance);
+  const IP_ADDRESS = "123.123.123.123";
+  const API_KEY = "api-key";
+
+  let postStub: SinonStub;
+
+  beforeEach(() => {
+    process.env.API_KEY = API_KEY;
+    process.env.FRONTEND_API_BASE_URL = "https://gov.uk";
+    postStub = sinon.stub(httpInstance.client, "post");
+  });
+
+  afterEach(() => {
+    postStub.reset();
+    delete process.env.API_KEY;
+    delete process.env.FRONTEND_API_BASE_URL;
+  });
+
+  it("successfully calls the API to log the user in or reauthenticate", async () => {
+    const axiosResponse = Promise.resolve({
+      data: {},
+      status: 200,
+      statusText: "OK",
+    });
+    postStub.resolves(axiosResponse);
+    req = createMockRequest(API_ENDPOINTS.LOG_IN_USER);
+    req.ip = IP_ADDRESS;
+    req.headers = {
+      "txma-audit-encoded": "audit encoded",
```

Comment:
```suggestion
      "txma-audit-encoded": "R21vLmd3QilNKHJsaGkvTFxhZDZrKF44SStoLFsieG0oSUY3aEhWRVtOMFRNMVw1dyInKzB8OVV5N09hOi8kLmlLcWJjJGQiK1NPUEJPPHBrYWJHP358NDg2ZDVc",
```

Reply:
Consider using this as its more representative of the actual value we expect in the header.  Make a constant and use in expectations.